security/pkg/nodeagent/caclient/providers/google/client.go

```diff
@@ -103,14 +103,15 @@ func (cl *googleCAClient) CSRSign(ctx context.Context, reqID string, csrPEM []by
 	ctx = metadata.NewOutgoingContext(ctx, out)
 	resp, err := cl.client.CreateCertificate(ctx, req)
 	if err != nil {
-		googleCAClientLog.Errorf("Failed to create certificate: %v", err)
+		googleCAClientLog.Errorf("Failed to create certificate: %v %v", err, req)
```

Comment:
"Failed to create certificate for request [%v]: %v"

Reply:
Can you just not log the request like this? We get a bunch of super obscure log messages because we do this. We shouldn't be logging raw go structures, especially ones that contain a huge CSR. Can you just log what we need?

Reply:
Added a Debugf with the detailed struct - the problem is that almost everything related to managed CA is close to impossible to troubleshoot, there are almost no checks at startup ( like the missing GKE_URL ) and the errors don't provide info about what went wrong. Looking at the request sometimes help. 

Using debug seems reasonable - it's off by default, and other systems ( K8S ) log full request/responses at debug level.